docs/source/common_issues.rst

```diff
@@ -180,6 +180,50 @@ not support ``sort()``) as a list and sort it in-place:
         # Type of x is List[int] here.
         x.sort()  # Okay!
 
+Invariance vs covariance
+------------------------
+
+Most mutable generic collections are invariant, as well all user defined
```

Comment:
This is a run-on sentence. Maybe reword as "Most mutable generic collections are invariant, and mypy considers all user-defined generic classes invariant by default (see :ref:`variance-of-generics` for motivation)."